src/LCRmeter-Metrohm_AutolabPGSTAT/LCRmeter-Metrohm_AutolabPGSTAT.ini

```diff
@@ -0,0 +1,5 @@
+[Autolab HardwareSetup]
+adk_path = C:\Program Files\Metrohm Autolab\Autolab SDK 2.1\Hardware Setup Files\Adk.x
+hardware_setup_files =
+    C:\Program Files\Metrohm Autolab\Autolab SDK 2.1\Hardware Setup Files\PGSTAT302N\HardwareSetup.AUT83940.xml,
```

Comment:
Should this go into a published driver if the hardware configuration file is specific to a certain customer?

Reply:
I'm not complaining about the template ini itself, just the chosen xml file - because this particular xml file is not part of the SDK installation.